proposals/ecosystem/index.bs

```diff
@@ -1610,9 +1627,438 @@ Each new [=Registered Data Instance=] must be linked to the
+that [=shape tree=] that have been explicitly associated with the
+[=Access Need=].
+
+Each [=Access Need=] has one or more [=access modes=], and a property that
+indicates the necessity of the [=Access Need=]; *required* or
+*optional*. For example, an [=Access Need=] for read access to
+[=shape tree=] `N` can be identified as a required item in the
+[=Access Need Group=].
+
+Because each [=Access Need=] is associated with a specific [=shape tree=],
+and [=shape trees=] provide both physical and/or virtual hierarchies, an
+[=Access Need=] can specify whether to recurse through the [=shape tree=]
+hierarchy. Other [=Access Needs=] in the same [=Access Need Group=] can be added
+to stop the recursion, or it can continue until it traverses through the
+remainder of the tree. This is helpful when dealing with more complex
+[=Access Needs=] like medical or financial records.
+
+Explaining *why* the access is needed is one of the most important
+elements when communicating [=Access Needs=]. [=Access Decorators=]
+explain why access is needed and how that access will be used for a given
+[=Access Need Group=], and every [=Access Need=] associated with it. This helps
+to ensure [=Agents=] make smart and informed authorization decisions.
+[=Access Decorators=] are designed to support different languages
+and accessibility needs.
+
+## Data Model ## {#authz-datamodel}
+
+An [=Access Need Group=] is represented by the class
+[eco:AccessNeedGroup](ecosystem.ttl#AccessNeedGroup) and can be invoked
+with the following properties:
+
+* [eco:requestsAccess](ecosystem.ttl#requestsAccess) - links to one or more
+    [=Access Needs=].
+* [eco:authenticatesAs](ecosystem.ttl#authenticatesAs) - [=Agent=] or
+    [eco:Pilot](ecosystem.ttl#Pilot).
+* [eco:accessNecessity](ecosystem.ttl#accessNecessity) - Can be
+    [eco:AccessRequired](ecosystem.ttl#AccessRequired) or
+    [eco:AccessOptional](ecosystem.ttl#AccessOptional)
+* [eco:hasAccessDecoratorIndex](ecosystem.ttl#hasAccessDecoratorIndex) -
+    links to an [=Access Decorator Index=] of [=Access Decorators=]
+
+<figure id="fig-access-need-group">
+  <figcaption>An [=Access Need Group=] linking to [=Access Needs=]</figcaption>
+  <pre highlight="turtle">
+  <#notebook-collaboration-group>
+    a eco:AccessNeedGroup ;
+    eco:accessNecessity eco:AccessRequired ;
+    eco:authenticatesAs &lt;https://nevernote.example/profile/id#agent&gt; ;
+    eco:requestsAccess &lt;#notebook&gt;, &lt;#note&gt; ;
+    eco:hasAccessDecoratorIndex &lt;#notebook-access-decorator-index&gt; .
+  </pre>
+</figure>
+
+An [=Access Need Group Decorator=] is represented by the class
+[eco:AccessNeedGroupDecorator](ecosystem.ttl#AccessNeedGroupDecorator),
+and can be invoked with the following properies:
+
+* [eco:hasAccessNeedGroup](ecosystem.ttl#hasAccessNeedGroup) - Links to
+    the [=Access Need Group=] being described
+* <a href="http://www.w3.org/2004/02/skos/core#prefLabel" class="vocab">
```

Comment:
Any particular reason to use skos `prefLabel` here, and not `rdfs:label` as is used elsewhere in solid spec?

Reply:
We currently use `skos:narrower` to create a UI hierarchy. Some generic SKOS tools might be able to do something useful with that, plus the multi-lang conventions for `skos:prefLabel` are better-established than for `rdfs:label`.